po/fr.po

```diff
@@ -38,7 +38,7 @@ msgstr ""
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:11
 msgid "For example:- \"spr`\" expands to \"Snippet Pixie rules!\""
-msgstr "Par exemple : - « spr` » s'étendra en « Snippet Pixie rules ! »"
+msgstr "Par exemple : - « rsp` » s'étendra en « Règles de Snippet Pixie ! »"
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:20
```

Comment:
Hey Nathan, don't suppose you mind fixing this translation too? I originally made a mistake and added "or welcome screen" to this line rather than where it is now, on the related "import" entry (line 22 of the associated xml.in file).

Reply:
You posted the wrong lines of code, but I think I understood and I just adjusted the translations